src/map.cpp

```diff
@@ -2000,9 +1995,7 @@ void MMVManip::initialEmerge(v3s16 blockpos_min, v3s16 blockpos_max,
 		u8 flags = 0;
 		MapBlock *block;
 		v3s16 p(x,y,z);
-		std::map<v3s16, u8>::iterator n;
-		n = m_loaded_blocks.find(p);
-		if(n != m_loaded_blocks.end())
+		if (m_loaded_blocks.count(p) > 0)
```

Comment:
For maps it might not be make a difference, but in general I find this is bad practice. `find` can stop as soon as a match is found. `count` has to scan the collection to count matching elements.
(Again, fine for maps.)

`if (m_loaded_blocks.find(p) != m_loaded_blocks.end())` is not much more verbose.

Reply:
I would use [`contains`](https://en.cppreference.com/w/cpp/container/map/contains) but that's c++20.
Anyway this operation is most common with maps or sets so I think it's okay.